raiden_installer/web.py

```diff
@@ -156,7 +163,33 @@ def _run_funding(self, configuration_file: RaidenConfigurationFile):
             self._send_status_update(f"Minting {transfer_token.ticker}")
             mint_tokens(w3, account, transfer_token)
 
+    def _run_create_wallet(self, **kw):
+        form = PasswordForm(passphrase1=kw.get("passphrase1"), passphrase2=kw.get("passphrase2"))
+        if form.validate():
+            self._send_status_update("Generating new wallet file for Raiden")
+            passphrase = form.data["passphrase1"].strip()
+            account = Account.create(passphrase=passphrase)
+            network_name = "mainnet"
```

Comment:
why not use the kw which is passed by the frontend for `network`

Reply:
good catch, this shouldn't be here anymore!

Reply:
ah, it turns out, the `network_name` is missing and needs to be passed around